mlir/test/Integration/Dialect/Arith/CPU/test-arith-expand-truncf-extf.mlir

```diff
@@ -0,0 +1,58 @@
+// Check various edge cases for truncf/extf ops involving f32 and f4e2m1 types.
+
+// RUN: mlir-opt %s --convert-vector-to-llvm \
+// RUN:             --convert-func-to-llvm \
+// RUN:             --arith-expand="include-f4e2m1=true" \
+// RUN:             --convert-arith-to-llvm | \
+// RUN: mlir-runner -e entry --entry-point-result=void \
+// RUN:             --shared-libs=%mlir_c_runner_utils | \
+// RUN: FileCheck %s --match-full-lines
```

Comment:
```suggestion
// RUN:   mlir-runner -e entry --entry-point-result=void \
// RUN:               --shared-libs=%mlir_c_runner_utils | \
// RUN:   FileCheck %s --match-full-lines
```

nit/optional: I like to indent these to make it easier to tell that these get the input from the pipe